### PR TITLE
clash-mini: Add version 0.1.6

### LINKED ISSUE #7920: [Request] Add Clash.Mini
### Information

**Name:** Clash.Mini

**Description:** A simple GUI for Clash.

**Homepage:** https://github.com/MetaCubeX/Clash.Mini

**Download link(s):** https://github.com/MetaCubeX/Clash.Mini/releases

**Some indication of popularity/repute:** https://github.com/MetaCubeX/Clash.Mini

### PR DESCRIPTION
**Pull request by bykenx**
[Clash.Mini](https://github.com/MetaCubeX/Clash.Mini) a simple GUI for [Clash](https://github.com/Dreamacro/clash).

Closes #7920

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
autoupdate hash isn't working. Try using some regex field there.

**Comment by bykenx** [comment]
> autoupdate hash isn't working. Try using some regex field there.

before commit, I has tested executing ` . \bin\checkver.ps1 clash-mini`, then the `hash` field is updated automatically, the hash file is a plain text only contains `sha256` hash data, I think there is no need to use `regex` to match the hash.

**Comment by rashil2000** [comment]
Can you paste the output of `.\bin\checkver.ps1 clash-mini -f` here?

**Comment by bykenx** [comment]
> Can you paste the output of `.\bin\checkver.ps1 clash-mini -f` here?

The logs show that the hash is computed from the file, I rechecked the hash file and found that there is a blank line at the end of the file, and the built-in regex `(^([a-fA-F0-9]+)$)` cannot match  a blank line

**Comment by rashil2000** [comment]
Yes, that's why I asked you to use custom regex. The builtin regex bug is fixed upstream (but not released yet).

File: bucket/clash-mini.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        "64bit": {
+            "url": "https://github.com/MetaCubeX/Clash.Mini/releases/download/v0.1.6/Clash.Mini_v0.1.6_x64.7z",
+            "hash": "716848db574808c38937669d74e868269281b86b03946cc7698405062c66dd63"
+        },
+        "32bit": {
+            "url": "https://github.com/MetaCubeX/Clash.Mini/releases/download/v0.1.6/Clash.Mini_v0.1.6_x86.7z",
+            "hash": "c14ce8714e268d6249a3eca49fe7f0f363efbff74644c66da544ded438be3345"
+        }
+    },
+    "bin": "Clash.Mini.exe",
```
Comment (rashil2000) [inline bucket/clash-mini.json:16]: Any commandline arguments?

Comment (bykenx) [reply bucket/clash-mini.json:16]: No args.

Comment (bykenx) [reply bucket/clash-mini.json:16]: > Any commandline arguments?

No args.

Comment (rashil2000) [reply bucket/clash-mini.json:16]: Then bin is not required

**Pull request merged**